### PR TITLE
Update types ScalarData,Manager

### PR DESCRIPTION
**Pull request by AhmedKorim**
Updated types, Metadata

### CODE REVIEW

File: packages/type-definitions/src/runtime.ts [hunk 1]
```diff
@@ -2,6 +2,8 @@ export default {
   rpc: {},
   types: {
     BlockWeights: 'Vec<u8>',
-    ChainId: 'u32'
+    ChainId: 'u32',
+    ScalarData: '[u8; 8]',
+    Manager: 'Vec<u8>'
```
Comment (shekohex) [inline packages/type-definitions/src/runtime.ts:7]: This should be a struct as defined in the anon node, I will try to fix this or I could help you to implement it today!

**Review by shekohex — CHANGES_REQUESTED** [review]

File: packages/type-definitions/src/runtime.ts (hunk 1 above)
Comment (AhmedKorim) [reply packages/type-definitions/src/runtime.ts:7]: Cool, can we merge it for now? I need to get the dApp withdrawal and deposit working today

Comment (shekohex) [reply packages/type-definitions/src/runtime.ts:7]: Ok, I will do a separate PR for it later today!

**Review by shekohex — APPROVED** [review]
I will do the missing types def and update it later today, looks good

**Pull request merged**